examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/main/paywalls/PaywallsScreen.kt

```diff
@@ -22,18 +22,27 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.revenuecat.paywallstester.SamplePaywalls
 import com.revenuecat.paywallstester.SamplePaywallsLoader
+import com.revenuecat.paywallstester.ui.theme.googleFont
 import com.revenuecat.purchases.CustomerInfo
 import com.revenuecat.purchases.Offering
 import com.revenuecat.purchases.models.StoreTransaction
 import com.revenuecat.purchases.ui.revenuecatui.PaywallDialog
 import com.revenuecat.purchases.ui.revenuecatui.PaywallDialogOptions
 import com.revenuecat.purchases.ui.revenuecatui.PaywallListener
+import com.revenuecat.purchases.ui.revenuecatui.fonts.CustomFontProvider
+import com.revenuecat.purchases.ui.revenuecatui.fonts.FontProvider
 
 @Composable
 fun PaywallsScreen(
     samplePaywallsLoader: SamplePaywallsLoader = SamplePaywallsLoader(),
 ) {
     var displayPaywallDialogOffering by remember { mutableStateOf<Offering?>(null) }
+    var fontProvider by remember { mutableStateOf<FontProvider?>(null) }
```

Comment:
At least in SwiftUI the way to do this would be to combine the state in a common type. Would that be too verbose in Kotlin?

Reply:
Well, we could definitely make a `sealed class` containing all this state, which should be cleaner 👍 Will do that change.

Reply:
Changed in https://github.com/RevenueCat/purchases-android/pull/1358/commits/7766689ef6edae5c94c6a01f29c5164d2dcaa2ab. Do you want to take another look @NachoSoto ?